database/games.js

```diff
@@ -0,0 +1,86 @@
+const PUBLISHERS = ['Electronic Arts', 'Square Enix', 'Nintendo',
+  'Sony', 'XSEED Games', 'Capcom', 'Activision Blizzard',
+  'Bandai Namco', 'Telltale Games', 'Ubisoft'];
+const GENRES = ['action', 'fps', 'rpg'];
+const MAX_PRICE = 65;
+const MIN_PRICE = 10;
+const MAX_OWNERSHIP = 100000;
+const MIN_OWNERSHIP = 10000;
+const MAX_RATING = 100;
+const MIN_RATING = 50;
+
+var incrementDate = function(date, daysToIncrement) {
+  var result = new Date(date);
+  result.setDate(result.getDate() + daysToIncrement);
+  return result;
+};
+
+var getRandomItemFromArray = function(array) {
+  var randomIndex = Math.floor(Math.random() * array.length);
+  return array[randomIndex];
+};
+
+var getRandomItemFromRange = function(low, high) {
+  return Math.floor(Math.random() * (high - low + 1) + low);
+};
+
+var createRandomGameObject = function(name, releaseDate) {
+  var publisher = getRandomItemFromArray(PUBLISHERS);
+  var genre = getRandomItemFromArray(GENRES);
+  var ownership = getRandomItemFromRange(MIN_OWNERSHIP, MAX_OWNERSHIP);
+  var rating = getRandomItemFromRange(MIN_RATING, MAX_RATING);
+  var price = getRandomItemFromRange(MIN_PRICE, MAX_PRICE);
+  var gameObj = {
+    name: name,
+    publisher: publisher,
+    releaseDate: releaseDate,
+    genre: genre,
+    rating: rating,
+    price: price,
+    ownership: ownership
+  };
+  return gameObj;
+};
+
+var createRandomGameArray = function(name, releaseDate) {
```

Comment:
@dfried514, is there a reason why you're wrapping the gameObj in an array? I don't see an obvious reason why from my brief review here, so I'm just curious.

Reply:
Hi Lam, I created arrays rather than objects because the connection.query('INSERT INTO games....') method requires that all field values be held in an array and not an object.  So creating objects and then converting back to array seemed inefficient.